### PR TITLE
add d_cahce cheet shhet

### PR DESCRIPTION
**Pull request by roman012285**

### CODE REVIEW

File: docs/cache/cheat_sheet.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+## N
+
+## O
+- **Offset** : The position of the data within a cache line, indicating the exact byte or word location.
+## P
+
+## Q
+
+## R
+- **Re-issue** : Action taken when an instruction or a data request cannot be immediately completed due to a cache miss or some other delay, and must be attempted again later. In that situation the request temporary removed from pipeline for other requests.
```
Comment (amichai-bd) [inline docs/cache/cheat_sheet.md:47]: Afaik, we are not "removing"it from pipe to allow others requests. We are "stalling'' the request on the re-issue buffer untill we finish handling rhe other requests that caused the backpressue. (Tq full, read miss)

File: docs/cache/cheat_sheet.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+- **Read modify write** : Process of modifying data by first reading it, update (modify) and write back. For example: If we want to update word in a cache line than we first read the cache line and merge it using merge buffer with the write data.
```
Comment (amichai-bd) [inline docs/cache/cheat_sheet.md:48]: Nope.. the merge buffer is not performing the read modified write in our design..

The read modified write takes place within the pipe for updating attributes in the tag array, and updatong data in the cach array

File: docs/cache/cheat_sheet.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+## S
+- **Set** : A group of cache lines where data from any given block of main memory can be placed, according to the cache's set-associativity.
```
Comment (amichai-bd) [inline docs/cache/cheat_sheet.md:51]: It's actually specific bit's within the address which are used to group the entire memory space into groups that can potentially be allocated next to each other (in the same "set")

File: docs/cache/cheat_sheet.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+## T
+- **Tag** : A portion of the physical memory address used to indicate the memory chunk ID from the far memory(FM).
```
Comment (amichai-bd) [inline docs/cache/cheat_sheet.md:53]: Note exactly.. the tag is the portion of the pa (physical address), but each cl has a unique set+tag. So the 'cl' I'd is actually both. Different cl can have the same rag. Different cl can have the same set. If both set and tag are the same for 2 cl - it's actually the same cl.

File: docs/cache/cheat_sheet.md
Comment (amichai-bd) [inline docs/cache/cheat_sheet.md:1]: Please review my comments

File: docs/cache/cheat_sheet.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
+## A
+- **Allocation** : The transaction(request) that enters the transaction queue (TQ)
```
[the remark this answers is not in the document]
Comment (roman012285) [reply docs/cache/cheat_sheet.md:2]: thanks, fixed!

File: docs/cache/cheat_sheet.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+## I
+
+## J
+
+## K
+
+## L
+
+## M
+- **MB** : Merge buffer. Buffer that located inside the Transaction queue entry(TQ) used to merge CL of 128 width with shorter data like word, half word and bytes.
```
[the remark this answers is not in the document]
Comment (roman012285) [reply docs/cache/cheat_sheet.md:33]: fixed!

File: docs/cache/cheat_sheet.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+## J
+
+## K
+
+## L
+
+## M
+- **MB** : Merge buffer. Buffer that located inside the Transaction queue entry(TQ) used to merge CL of 128 width with shorter data like word, half word and bytes. 
+- **Modified** : Modified data is data that been changed inside the cache and has not been stored back into far memory(FM) causing the far memory(FM) to be not updated.
```
[the remark this answers is not in the document]
Comment (roman012285) [reply docs/cache/cheat_sheet.md:34]: done

**Pull request merged**